docs/machine-learning/tutorials/object-detection-model-builder.md

```diff
@@ -55,68 +55,9 @@ This sample creates a C# .NET Core console application that detects stop signs i
-1. Change the **Target Connection** to *StopSignImages* as well. Your **Project Settings** should now look similar to this screenshot:
-
-    ![VoTT Project Settings Dialog](./media/object-detection-model-builder/vott-new-project.png)
-
-1. Select **Save Project**.
-
-### Add tag and label images
-
-You should now see a window with preview images of all the training images on the left, a preview of the selected image in the middle, and a **Tags** column on the right. This screen is the **Tags editor**.
-
-1. Select the first (plus-shaped) icon in the **Tags** toolbar to add a new tag.
-
-    ![VoTT New Tag Icon](./media/object-detection-model-builder/vott-new-tag-icon.png)
-
-1. Name the tag "Stop-Sign" and hit <kbd>Enter</kbd> on your keyboard.
-
-    ![VoTT New Tag](./media/object-detection-model-builder/vott-new-tag.png)
-
-1. Click and drag to draw a rectangle around each stop sign in the image. If the cursor does not let you draw a rectangle, try selecting the **Draw Rectangle** tool from the toolbar on the top, or use the keyboard shortcut <kbd>R</kbd>.
-1. After drawing your rectangle, select the **Stop-Sign** tag that you created in the previous steps to add the tag to the bounding box.
-1. Click on the preview image for the next image in the dataset and repeat this process.
-1. Continue steps 3-4 for every stop sign in every image.
-
-    ![VoTT Annotating Images](./media/object-detection-model-builder/vott-annotating.gif)
-
-### Export your VoTT JSON
-
-Once you have labeled all of your training images, you can export the file that will be used by Model Builder for training.
-
-1. Select the fourth icon in the left toolbar (the one with the diagonal arrow in a box) to go to the **Export Settings**.
-1. Leave the **Provider** as *VoTT JSON*.
-1. Change the **Asset State** to *Only tagged Assets*.
-1. Uncheck **Include Images**. If you include the images, then the training images will be copied to the export folder that is generated, which is not necessary.
-1. Select **Save Export Settings**.
-
-    ![VoTT Export Settings](./media/object-detection-model-builder/vott-export.png)
-
-1. Go back to the **Tags editor** (the second icon in the left toolbar shaped like a ribbon). In the top toolbar, select the **Export Project** icon (the last icon shaped like an arrow in a box), or use the keyboard shortcut <kbd>Ctrl</kbd>+<kbd>E</kbd>.
-
-    ![VoTT Export Button](./media/object-detection-model-builder/vott-export-button.png)
-
-This export will create a new folder called *vott-json-export* in your *Stop-Sign-Images* folder and will generate a JSON file named *StopSignObjDetection-export* in that new folder. You will use this JSON file in the next steps for training an object detection model in Model Builder.
+> If you want to skip the data labeling steps, you can [download this version of the dataset](https://aka.ms/mlnet-object-detection-tutorial-assets) and continue this tutorial.
```

Comment:
1. Remove references to VOTT tagging how-to guide from here.
2. Since the dataset is already tagged just instruct users to download the dataset.
3. Add a description of the dataset: 
     - Images are the raw images used for training
     - *-asset.json files contain bounding box information for the respective image.
     - StopSignObjDetection-export.json file has the references to the image files and bounding box information from asset files. It's an aggregate of all the asset files. This is the file used to load the data for training in Model Builder.